src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    int MIN_QUANTITY = 0;
+    void applyOperation(FruitTransaction transaction);
+
+    default Integer getCurrentBalance(FruitTransaction transaction) {
```

Comment:
Avoid providing default implementations in interfaces that are related to the state of the application, such as database operations. This can lead to a breach of the Interface Segregation Principle, one of the SOLID principles, as clients might be forced to depend on methods they do not use.